samples/BenchmarkDotNet.Samples/project.json

```diff
@@ -33,22 +33,35 @@
         }
       }
     },
-    "netcoreapp1.1": {
+    //"netcoreapp1.1": {
+    //  "buildOptions": {
+    //    "define": [ "CORE" ]
+    //  },
+    //  "dependencies": {
+    //    "Microsoft.NETCore.App": {
+    //      "type": "platform",
+    //      "version": "1.1.0"
+    //    },
+    //    "System.ComponentModel.EventBasedAsync": "4.0.11"
+    //  }
+    //},
+    "uap10.0": {
       "buildOptions": {
-        "define": [ "CORE" ]
+        "define": [ "CORE", "UAP" ]
       },
       "dependencies": {
-        "Microsoft.NETCore.App": {
-          "type": "platform",
-          "version": "1.1.0"
-        },
-        "System.ComponentModel.EventBasedAsync": "4.0.11"
+        "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
+        "System.Threading.Tasks": "4.3.0"
       }
     }
   },
   "dependencies": {
     "BenchmarkDotNet": {
       "target": "project"
     }
+  },
+  "runtimes": {
```

Comment:
we can not support only hardcoded list of runtimes, but you probably know that

Is it possible to set the runtimes per target framework?